qa/build.gradle

```diff
@@ -170,10 +170,10 @@ task unZipBwcMlCommonsPlugin(type: Zip) {
     }
 }
 
-// Task to unzip knn plugin from archive
-task unZipBwcKnnPlugin(type: Zip) {
+// Task to zip knn plugin from archive
+task zipBwcKnnPlugin(type: Zip) {
     dependsOn "pullKnnBwcPlugin"
-    dependsOn "unZipBwcMlCommonsPlugin"
+    dependsOn "zipBwcMlCommonsPlugin"
     from(Path.of(tmp_dir.absolutePath, "opensearch-knn"))
     destinationDirectory = tmp_dir
     archiveFileName = "opensearch-knn-${neural_search_bwc_version_no_qualifier}.zip"
```

Comment:
Do you know where is this archiveFileName used?

Reply:
Used to define file for plugin installation.

Reply:
You can check AbstractArchiveTask for gradle: https://docs.gradle.org/current/dsl/org.gradle.api.tasks.bundling.AbstractArchiveTask.html